dataInterface/movies.js

```diff
@@ -118,13 +141,35 @@ module.exports.updateById = async (movieId, newObj) => {
   const result = await movies.updateOne(filter, updateRules);
 
   if(result.modifiedCount != 1){
-    return {error: `Something went wrong. ${result.modifiedCount} movies were updated. Please try again.`}
+    return {error: `Something went wrong. ${result.modifiedCount} movies were not updated. Please try again.`}
```

Comment:
The original error message is correct. The `result.modifiedCount` is how many movies were updated. Our expectation is that we'd fall into this condition if the value was 0, but if it was something like 2, we'd really want to communicate that because it would be very weird.